### DISCUSSION TITLE
Really Weird NoSuchField Minecraft Error, how to fix? NO LOGIC BEHIND???

### DISCUSSION THREAD

**Discussion by copyandexecute** [FabricMC/fabric-api#3818] [Mod Dev Support]
### Description

![grafik](https://github.com/FabricMC/fabric/assets/58366920/360f9c20-e4b6-48f5-97d6-54ee6ff8ec80)
Repo: https://github.com/HGLabor/hero-modules

**Comment by copyandexecute** [comment]
https://github.com/HGLabor/hero-modules/blob/117d6614d0215c262ec83d61d33b552dcf50867e/ffa-server/src/main/kotlin/gg/norisk/ffa/server/FFAServer.kt#L21 heres the code

**Reply by NoRiskHelper** [reply]
Fehler akzeptieren und um verschiedene Dinge kümmern wie Freunde

**Comment by Chocohead** [comment]
You'll need to disable Loom's multi-project optimisation by removing [this line](https://github.com/HGLabor/hero-modules/blob/117d6614d0215c262ec83d61d33b552dcf50867e/gradle.properties#L4)
